Orange/widgets/visualize/owscatterplot.py

```diff
@@ -195,9 +195,14 @@ def __init__(self):
             box, self, "graph.attr_size", label="Size:",
             emptyString="(Same size)", callback=self.graph.update_sizes,
             **common_options)
+        self.sl_point_size = gui.hSlider(box, self, "graph.point_width", label="Symbol size:   ",
+                                          minValue=1, maxValue=20, step=1, createLabel=False,
```

Comment:
Sorry for my nitpicking: can you remove a space from this line and the next? Also, can you break the line to fit to 80 columns?

We also don't need `self.sl_point_size`.